tests/features/metadata.feature

```diff
@@ -8,13 +8,13 @@ Feature: As a SP or IdP
     And the response should match xpath '//md:SingleSignOnService[@Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect"]'
     And the response should match xpath '//md:SingleSignOnService[@Location="https://gateway.stepup.example.com/authentication/single-sign-on"]'
 
-  Scenario: View the SecondFactor metadata of the Gateway
+  Scenario: View the SecondFactor of the Gateway
```

Comment:
Oops: I think this should read: `View the second factor only metadata of the Gateway`